app/views/members/_form.html.erb

```diff
@@ -98,11 +98,16 @@
       <dd><%= f.text_field :interests, class: "wide-input" %></dd>
     </dl>
 
+    <dl>
+        <dt><%= f.label :prefers_full_name, "Prefer full name:" %></dt>
```

Comment:
"Prefer full name" does not give a lot of context, a la "textable" and "social textable"... how about "Show real names of members" or similar?